### ISSUE TITLE
Failing migration between Grafana v6.6.2 and v6.7.0

### ISSUE THREAD

**Issue by TimShilov** [grafana/grafana#22925]
**What happened**:
We updated Grafana to v6.7.0 (it was v6.6.2 before) and now `grafana-server` is failing to start because of failing migration. Before the update it was working perfectly fine.
Please tell me how can I overcome that to bring it back up.

Here's the logs:
```
Mar 20 13:01:01 affluent-hub grafana-server[32517]: t=2020-03-20T13:01:01+0000 lvl=info msg="Starting Grafana" logger=server version=6.7.0 commit=c4656a885d branch=HEAD compiled=2020-03-19T11:27:02+0000
Mar 20 13:01:01 affluent-hub grafana-server[32517]: t=2020-03-20T13:01:01+0000 lvl=info msg="Config loaded from" logger=settings file=/usr/share/grafana/conf/defaults.ini
Mar 20 13:01:01 affluent-hub grafana-server[32517]: t=2020-03-20T13:01:01+0000 lvl=info msg="Config loaded from" logger=settings file=/etc/grafana/grafana.ini
Mar 20 13:01:01 affluent-hub grafana-server[32517]: t=2020-03-20T13:01:01+0000 lvl=info msg="Config overridden from command line" logger=settings arg="default.paths.data=/var/lib/grafana"
Mar 20 13:01:01 affluent-hub grafana-server[32517]: t=2020-03-20T13:01:01+0000 lvl=info msg="Config overridden from command line" logger=settings arg="default.paths.logs=/var/log/grafana"
Mar 20 13:01:01 affluent-hub grafana-server[32517]: t=2020-03-20T13:01:01+0000 lvl=info msg="Config overridden from command line" logger=settings arg="default.paths.plugins=/var/lib/grafana/plugins"
Mar 20 13:01:01 affluent-hub grafana-server[32517]: t=2020-03-20T13:01:01+0000 lvl=info msg="Config overridden from command line" logger=settings arg="default.paths.provisioning=/etc/grafana/provisioning"
Mar 20 13:01:01 affluent-hub grafana-server[32517]: t=2020-03-20T13:01:01+0000 lvl=info msg="Path Home" logger=settings path=/usr/share/grafana
Mar 20 13:01:01 affluent-hub grafana-server[32517]: t=2020-03-20T13:01:01+0000 lvl=info msg="Path Data" logger=settings path=/var/lib/grafana
Mar 20 13:01:01 affluent-hub grafana-server[32517]: t=2020-03-20T13:01:01+0000 lvl=info msg="Path Logs" logger=settings path=/var/log/grafana
Mar 20 13:01:01 affluent-hub grafana-server[32517]: t=2020-03-20T13:01:01+0000 lvl=info msg="Path Plugins" logger=settings path=/var/lib/grafana/plugins
Mar 20 13:01:01 affluent-hub grafana-server[32517]: t=2020-03-20T13:01:01+0000 lvl=info msg="Path Provisioning" logger=settings path=/etc/grafana/provisioning
Mar 20 13:01:01 affluent-hub grafana-server[32517]: t=2020-03-20T13:01:01+0000 lvl=info msg="App mode production" logger=settings
Mar 20 13:01:01 affluent-hub grafana-server[32517]: t=2020-03-20T13:01:01+0000 lvl=info msg="Writing PID file" logger=server path=/var/run/grafana/grafana-server.pid pid=32517
Mar 20 13:01:01 affluent-hub grafana-server[32517]: t=2020-03-20T13:01:01+0000 lvl=info msg="Initializing SqlStore" logger=server
Mar 20 13:01:01 affluent-hub grafana-server[32517]: t=2020-03-20T13:01:01+0000 lvl=info msg="Connecting to DB" logger=sqlstore dbtype=mysql
Mar 20 13:01:01 affluent-hub grafana-server[32517]: t=2020-03-20T13:01:01+0000 lvl=info msg="Starting DB migration" logger=migrator
Mar 20 13:01:01 affluent-hub grafana-server[32517]: t=2020-03-20T13:01:01+0000 lvl=info msg="Executing migration" logger=migrator id="Add time_from to the report"
Mar 20 13:01:01 affluent-hub grafana-server[32517]: t=2020-03-20T13:01:01+0000 lvl=eror msg="Executing migration failed" logger=migrator id="Add time_from to the report" error="Error 1064: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NULL' at line 1"
Mar 20 13:01:01 affluent-hub grafana-server[32517]: t=2020-03-20T13:01:01+0000 lvl=eror msg="Exec failed" logger=migrator error="Error 1064: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NULL' at line 1" sql="alter table `report` ADD COLUMN `time_from` VARCHAR CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NULL "
Mar 20 13:01:01 affluent-hub grafana-server[32517]: t=2020-03-20T13:01:01+0000 lvl=eror msg="Server shutdown" logger=server reason="Service init failed: Migration failed err: Failed to roll back transaction due to error: %!s(<nil>): Error 1064: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NULL' at line 1"
Mar 20 13:01:01 affluent-hub systemd[1]: grafana-server.service: main process exited, code=exited, status=1/FAILURE
Mar 20 13:01:01 affluent-hub systemd[1]: Failed to start Grafana instance.
Mar 20 13:01:01 affluent-hub systemd[1]: Unit grafana-server.service entered failed state.
Mar 20 13:01:01 affluent-hub systemd[1]: grafana-server.service failed.
Mar 20 13:01:01 affluent-hub systemd[1]: grafana-server.service holdoff time over, scheduling restart.
Mar 20 13:01:01 affluent-hub systemd[1]: Stopped Grafana instance.
```

**What you expected to happen**:
I expected migration to execute without errors and Grafana continue to work.

**How to reproduce it (as minimally and precisely as possible)**:
Update Grafana from v6.6.2 to v6.7.0.

**Anything else we need to know?**:
 I don't know if there's any other parts to this failure, we changed nothing related to Grafana lately.

**Environment**:
- Grafana version: v6.7.0
- Data source type & version: unrelated
- OS Grafana is installed on: CentOS 7 (installed from https://packages.grafana.com/enterprise/rpm)
- User OS & Browser: unrelated
- Grafana plugins: Image Renderer
- Others: -

**Comment by torkelo** [comment]
fix coming in 6.7.1 (releasing now)

**Comment by TimShilov** [comment]
Just updated to 6.7.1 and it's working now!
Thank you! 👍
